src/styling/projects-page-styling.css

```diff
@@ -31,12 +96,52 @@
 }
 
 .scrolling {
-  height: 85vh;
+  height: 90vh;
   overflow-y: scroll;
   overflow: visible;
+
+
 }
 
-.card-row {
-  margin-top: 10px;
+.project-col {
+  background-color: var(--main-grey);
+  border-radius: 10px;
+  padding: 20px;
+  margin-right: 15px;
+}
+
+.close-btn {
+    padding: 15px;
+    padding-left: 20px;
+    padding-right: 20px;
```

Comment:
nit: instead of having all 3 padding rules you can just do `padding: 15px 20px`

Reply:
oh good to know! fixed for all